clients/cloud/rust/src/producer.rs

```diff
@@ -0,0 +1,44 @@
+extern crate futures;
+extern crate rdkafka;
+
+use futures::future::Future;
+use rdkafka::error::KafkaError;
+use rdkafka::message::OwnedMessage;
+use rdkafka::producer::{FutureProducer, FutureRecord};
+use std::boxed::Box;
+
+mod utils;
+
+fn log_produce_result(result: Result<(i32, i64), (KafkaError, OwnedMessage)>) -> Result<(), ()> {
+    result
+        .and_then(|(p, o)| {
+            println!("produced to partition {} with offset {}", p, o);
```

Comment:
For consistency with the other client examples, can this print topic name as well?  Specifically, this is what the other clients print:

```bash
Produced record to topic test1 partition [0] @ offset 0
...
```